### PR TITLE
Pioneer 20a gliders

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: CP15MOAS-GL387/CP15MOAS-GL387_R00001_ingest.csv
Comment (reedan88) [inline CP15MOAS-GL387/CP15MOAS-GL387_R00001_ingest.csv:1]: Looks good.

File: CP15MOAS-GL379/CP15MOAS-GL379_D00002_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL379/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-GL379-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL379/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-GL379-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL379/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-GL379-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL379/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-GL379-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL379/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-GL379-05-PARADM000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP15MOAS-GL379/CP15MOAS-GL379_D00002_ingest.csv:6]: looks good - data on raw data repo

File: CP15MOAS-GL387/CP15MOAS-GL387_D00002_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL387/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-GL387-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL387/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-GL387-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL387/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-GL387-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL387/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-GL387-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP15MOAS-GL387/D00002/merged-from-glider/cp_*.mrg,CP15MOAS-GL387-05-PARADM000,telemetered,Available,
```
Comment (rtravis-whoi) [inline CP15MOAS-GL387/CP15MOAS-GL387_D00002_ingest.csv:6]: looks good - data on raw data repo

File: CP15MOAS-GL387/CP15MOAS-GL387_R00001_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+#mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP15MOAS-GL387/R00001/merged/cp_*.mrg,CP15MOAS-GL387-02-FLORTM000,recovered_host,Expected,Glider re-deployed without opening; data expected after D00002 recovery
+#mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP15MOAS-GL387/R00001/merged/cp_*.mrg,CP15MOAS-GL387-03-CTDGVM000,recovered_host,Expected,Glider re-deployed without opening; data expected after D00002 recovery
+#mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP15MOAS-GL387/R00001/merged/cp_*.mrg,CP15MOAS-GL387-04-DOSTAM000,recovered_host,Expected,Glider re-deployed without opening; data expected after D00002 recovery
+#mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP15MOAS-GL387/R00001/merged/cp_*.mrg,CP15MOAS-GL387-05-PARADM000,recovered_host,Expected,Glider re-deployed without opening; data expected after D00002 recovery
```
Comment (rtravis-whoi) [inline CP15MOAS-GL387/CP15MOAS-GL387_R00001_ingest.csv:7]: looks good - readme file in raw data repo

**Review by rtravis-whoi — APPROVED** [review]

**Pull request merged**